sql/02-bde_control_functions.sql.in

```diff
@@ -1966,7 +1966,12 @@ BEGIN
         RAISE INFO '%', v_task;
 
         PERFORM bde_control._bde_GetExclusiveLock(p_upload,v_bdetable);
-        v_sql := 'DROP TABLE ' || v_bdetable || ' CASCADE';
+
+        v_sql := 'PERFORM table_version.ver_disable_versioning( ' || v_bdetable || ')';
```

Comment:
Can we please do a check to ensure the table version support is present before calling this API.